doc-src/templates/default/fulldoc/html/full_list.erb

```diff
@@ -28,7 +28,7 @@
         </div>
 
         <!-- Add label for search -->
-        <div id="search">
+        <div id="search" >
```

Comment:
Is the space significant?

Reply:
No. I was doing something here before. I've removed it.